frontend/index.html

```diff
@@ -7,22 +7,36 @@
 		<link
 			rel="apple-touch-icon"
 			sizes="180x180"
-			href="/apple-touch-icon.png"
+			href="./src/apple-touch-icon.png"
 		/>
+		<link rel="apple-touch-icon" href="./src/logo192.png" />
```

Comment:
Reordered these and found out we seem to have 2 apple-touch-icons at very similar sizes. Maybe we could remove one?